floodscan_susceptibliity_target_pipeline.R

```diff
@@ -246,7 +314,53 @@ list(
       split(.$adm0_en) %>%
       imap(\(dft, country_name){
         dft %>%
-          select(adm0_en, adm1_en, adm2_en, mean_exposed, total_pop, pct_exposed) %>%
+          dplyr::select(adm0_en, adm1_en, adm2_en, mean_exposed, total_pop, pct_exposed) %>%
+          group_by(adm1_en) %>%
+          gt() %>%
+          cols_hide(columns = "adm0_en") %>%
+          cols_label(
+            # adm0_en = "Country",
+            adm1_en = "Region",
+            adm2_en = "District",
+            mean_exposed = "Average population exposed",
+            total_pop = "Total population",
+            pct_exposed = "Percent population exposed"
+          ) %>%
+          fmt_number(
+            columns = c(
+              "mean_exposed",
+              "total_pop",
+              "pct_exposed"
+            ),
+            decimals = 0, n_sigfig = 3
+          ) %>%
+          fmt_percent(columns = pct_exposed, decimals = 0) %>%
+          data_color(
+            columns = c(
+              "mean_exposed",
+              "total_pop",
+              "pct_exposed"
+            ),
+            method = "numeric",
+            palette = "YlOrRd",
+            reverse = F
+          ) %>%
+          tab_options(
+            row_group.as_column = TRUE
+          ) %>%
+          tab_header(
+            title = paste0("Approximate population susceptible to flooding ", country_name)
+          )
+      })
+  ),
+  tar_target(
+    name = lgt_eth_ssn_adm2_bin_method,
```

Comment:
table is missing row identifier. Some of the row numbers represent AMJ numbers while others do not, but its not clearly marked in the gt table